src/components/list/list.css

```diff
@@ -1,13 +0,0 @@
-.artsy-list{
-    list-style: none;
```

Comment:
can create an additional class for list-none so that it gives the users more options